internal/oci/runtime_syscall_exec.go

```diff
@@ -41,3 +53,22 @@ func (r syscallExec) Exec(args []string) error {
 func (r syscallExec) String() string {
 	return "exec"
 }
+
+// Escape1 escapes shell metacharacters in a single command-line argument.
+func Escape1(arg string) string {
```

Comment:
theres' no need for this func to be public, also the name could be something more saying than `escape1`